scripts/tests/py/test_metadata.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from metadata import Metadata
+from metadata import Metadata_Reader
+
+
+class TestMetadataReader(unittest.TestCase):
+    # build the test Metadata_Reader object
+    #defines expected arguments associated with runs
+    def setUp(self):
+
+        # build the reader object
+        self.reader=Metadata_Reader("/src/python_testing", "/scripts/tests/py/env.yaml")
+        self.actual_runs_metadata = self.reader.get_runs_metadata()
+        self.expected_runs_metadata = {}
+
+        self.expected_runs_metadata["/src/python_testing/TC_RR_1_1.py+run1"] = Metadata(app="out/linux-x64-all-clusters-ipv6only-no-ble-no-wifi-tsan-clang-test/chip-all-clusters-app",
+                   discriminator=1234, py_script_path="/src/python_testing/TC_RR_1_1.py", run="run1", passcode=20202021)
```

Comment:
Please use test files for testing, not real tests. There should be separate test-only files (and maybe don't name them py, just txt for now) that validates different variants of items.

Reply:
These loads should be part of test case runs and not part of the setup. At most, the setup should prepare the reader.